nixos/modules/services/web-servers/stargazer.nix

```diff
@@ -21,143 +20,174 @@ let
 
-    listen = lib.mkOption {
+    listen = mkOption {
       type = types.listOf types.str;
       default = [ "0.0.0.0" ] ++ optional config.networking.enableIPv6 "[::0]";
       defaultText = literalExpression ''[ "0.0.0.0" ] ++ lib.optional config.networking.enableIPv6 "[::0]"'';
       example = literalExpression ''[ "10.0.0.12" "[2002:a00:1::]" ]'';
-      description = lib.mdDoc ''
+      description = mdDoc ''
         Address and port to listen on.
       '';
     };
 
-    connectionLogging = lib.mkOption {
+    connectionLogging = mkOption {
       type = types.bool;
       default = true;
-      description = lib.mdDoc "Whether or not to log connections to stdout.";
+      description = mdDoc "Whether or not to log connections to stdout.";
     };
 
-    ipLog = lib.mkOption {
+    ipLog = mkOption {
       type = types.bool;
       default = false;
-      description = lib.mdDoc "Log client IP addresses in the connection log.";
+      description = mdDoc "Log client IP addresses in the connection log.";
     };
 
-    ipLogPartial = lib.mkOption {
+    ipLogPartial = mkOption {
       type = types.bool;
       default = false;
-      description = lib.mdDoc "Log partial client IP addresses in the connection log.";
+      description = mdDoc "Log partial client IP addresses in the connection log.";
     };
 
-    requestTimeout = lib.mkOption {
+    requestTimeout = mkOption {
       type = types.int;
       default = 5;
-      description = lib.mdDoc ''
+      description = mdDoc ''
         Number of seconds to wait for the client to send a complete
         request. Set to 0 to disable.
       '';
     };
 
-    responseTimeout = lib.mkOption {
+    responseTimeout = mkOption {
       type = types.int;
       default = 0;
-      description = lib.mdDoc ''
+      description = mdDoc ''
         Number of seconds to wait for the client to send a complete
         request and for stargazer to finish sending the response.
         Set to 0 to disable.
       '';
     };
 
-    store = lib.mkOption {
+    store = mkOption {
       type = types.path;
       default = /var/lib/gemini/certs;
-      description = lib.mdDoc ''
+      description = mdDoc ''
         Path to the certificate store on disk. This should be a
         persistent directory writable by Stargazer.
       '';
     };
 
-    certOrg = lib.mkOption {
+    certOrg = mkOption {
       type = types.str;
       default = "stargazer";
-      description = lib.mdDoc ''
+      description = mdDoc ''
         The name of the organization responsible for the X.509
         certificate's /O name.
       '';
     };
 
-    genCerts = lib.mkOption {
+    genCerts = mkOption {
       type = types.bool;
       default = true;
-      description = lib.mdDoc ''
+      description = mdDoc ''
         Set to false to disable automatic certificate generation.
         Use if you want to provide your own certs.
       '';
     };
 
-    regenCerts = lib.mkOption {
```

Comment:
Those cosmetic changes, put them in another commit.

Better yet, undo them. `with lib;` is a plague we should remove from the codebase.

Reply:
Done. I've broken it out into 2 commits and remove `with lib;`.